test/model.test.ts

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 import { Enforcer } from '../src/enforcer';
+import * as _ from 'lodash';
```

Comment:
I have already fixed this issue in this commit: https://github.com/casbin/node-casbin/commit/01c735be548f4c0cd18ab1bb4aa2b00bb7d5f5f5, please remove it.